packages/core/admin/admin/src/components/GuidedTour/reducer.js

```diff
@@ -17,6 +17,7 @@ export const initialState = {
       success: false,
     },
   },
+  guidedTourVisibility: false,
```

Comment:
It's better for boolean to prefix them by `is, has, does, should` 

Could you rename this to `isGuidedTourVisible` ?